lib/index.ts

```diff
@@ -8,4 +8,29 @@
 import * as Temporal from './temporal';
 import * as Intl from './intl';
 import { toTemporalInstant } from './legacydate';
+
+// Work around https://github.com/babel/babel/issues/2025.
+const types = [
+  Temporal.Instant,
+  Temporal.Calendar,
+  Temporal.PlainDate,
+  Temporal.PlainDateTime,
+  Temporal.Duration,
+  Temporal.PlainMonthDay,
+  // Temporal.Now,
```

Comment:
Either remove, or add a comment explaining why Now isn't also not-writeable?

Reply:
I think it's because Now doesn't have its own prototype, it's a plain object, but agreed a comment would be good

Reply:
Yep, that's why it's omitted. I'll add a comment and merge.